.circleci/local_publish_helpers.sh

```diff
@@ -276,11 +276,10 @@ function runE2eTest {
 
     if [ -f  $FAILED_TEST_REGEX_FILE ]; then
         # read the content of failed tests
-        failedTests=$(<$FAILED_TEST_REGEX_FILE)
-        # adding --force-exit per https://github.com/facebook/jest/issues/9473
-        yarn run e2e --force-exit --detectOpenHandles --maxWorkers=3 $TEST_SUITE -t "$failedTests"
+        failedTests=$(<$FAILED_TEST_REGEX_FILE)=
+        yarn run e2e --no-cache --maxWorkers=3 $TEST_SUITE -t "$failedTests"
     else
-        yarn run e2e --force-exit --detectOpenHandles --maxWorkers=3 $TEST_SUITE
```

Comment:
--detectOpenHandles implicitly forces tests to run in band (in series) so we must remove this to allow multi core utilization